javascript/code-challenges/queueWithStacks/queue-with-stacks.js

```diff
@@ -45,4 +45,6 @@ class PseudoQueue{
   }
 
   // dequeue()
-}
\ No newline at end of file
+}
+
+module.exports = Queue;
```

Comment:
It looks like this is causing an error, I am getting back: `ReferenceError: Queue is not defined`